docs/python/reference/dbos-class.md

```diff
@@ -0,0 +1,61 @@
+---
+sidebar_position: 1
+title: DBOS Class
+---
+
+The DBOS class is a singleton&mdash;you may only instantiate it once in a program's lifetime.
+Here, we document its constructor and lifecycle methods.
+Decorators are documented [here](./decorators.md) and context methods and variables [here](./contexts.md).
+
+### class dbos.DBOS
+
+```python
+DBOS(
+    *,
+    fastapi: Optional[FastAPI] = None,
+    config: Optional[ConfigFile] = None,
+)
+```
+
+**Parameters:**
+- `fastapi`: If your application is using FastAPI, the `FastAPI` object. If this is passed in, DBOS automatically calls `launch` when FastAPI is fully initialized. DBOS also adds to all routes a middleware that enables [tracing](../tutorials/logging-and-tracing.md#tracing) through FastAPI HTTP endpoints.
```

Comment:
Provide a pointer to "launch". It's unclear which launch it calls, FastAPI or itself.